### DISCUSSION TITLE
Sumatra PDF - Hand

### DISCUSSION THREAD

**Discussion by WAAH1** [sumatrapdfreader/sumatrapdf#2827] [Q&A]
hello,

i would like to lock the navigation hand on the sumatra PDF viewer, unfortunately, as soon as there is  a text, the hand become a selection...

is anyone can help me please ?

thanks !

**Comment by GitHubRulesOK** [comment]
Left click is select  shifted left or right mouse is grab, so you need to reverse buttons or shift to not select items or drag by use margins or other void spaces.

Discussion here suggests add AutoHotKey control https://github.com/sumatrapdfreader/sumatrapdf/issues/299

**Reply by WAAH1** [reply]
thanks a lot! 
indeed i shoud right click then
by any chance, do you know if i can reverse buttons on the mouse ? in order to keep the left click to grab the page (even if when there is some text bloc), this is more intuitive for me  

thanks

**Reply by GitHubRulesOK** [reply]
that reversal is a windows system command I think i showed **main.cpl** shortcut in #299

**Reply by GitHubRulesOK** [reply]
You can use mouse tools to change behavior per application
![image](https://user-images.githubusercontent.com/12754913/181122037-895ca672-5d86-4fe9-a817-0e7686663001.png)

**Reply by WAAH1** [reply]
excellent !
thanks

**Reply by GitHubRulesOK** [reply]
just beware that if you change Left Mouse Button you should also ***change something else to replace it*** otherwise you cant select menus etc. :-)
